AzureRmStorageTableCoreHelper.psm1

```diff
@@ -595,6 +633,9 @@ function Get-AzTableRow
 		$TableQuery.FilterString = $Filter
 	}
 
+	# Set number of rows to return ($Top is null if unset, in which case there's no limit.)
+	$TableQuery.TakeCount = $Top

```

Comment:
Please add a null check before assigning the TakeCount property.

Reply:
Done.